Source/Mosa.UnitTests/Primitive/StringTests.cs

```diff
@@ -78,4 +78,76 @@ public static bool NotEqual1()
 
 		return string.Equals(a, b);
 	}
+
+	[MosaUnitTest]
+	public static bool SubstringTest1()
+	{
+		return "abc".Substring(1).Equals("bc");
+	}
+
+	[MosaUnitTest]
+	public static bool SubstringTest2()
+	{
+		return "abcdef".Substring(3).Equals("def");
+	}
+
+	[MosaUnitTest]
+	public static bool SubstringTest3() {
+		return "abcdef".Substring(0, 3).Equals("abc");
+	}
+
+	[MosaUnitTest]
+	public static bool SubstringTest4()
+	{
+		return "abcdef".Substring(2, 1).Equals("c");
+	}
+
+	[MosaUnitTest]
+	public static bool InsertTest1()
+	{
+		return "aaa".Insert(0, "bbb").Equals("aaabbb");
```

Comment:
Shouldn't this equal to "bbbaaa" instead?

Reply:
Oops, yes it should